docs/install-github.md

```diff
@@ -149,10 +162,16 @@
            // [Mandatory]
            // API endpoint to GitHub enterprise server
            JF_GIT_API_ENDPOINT = ""
-           // Uncomment the below options if you'd like to use them.
+           // [OPTIONAL]
+           // Frogbot Management repository in which the frogbot-config.yml resides
+           JF_GIT_REPO: ""
+           // [OPTIONAL]
+           // Frogbot Management repository branch in which the frogbot-config.yml resides
+           JF_GIT_BASE_BRANCH: ""
            // [Mandatory if JF_USER and JF_PASSWORD are not provided]
            // JFrog access token with 'read' permissions for Xray
            // JF_ACCESS_TOKEN= credentials("JF_ACCESS_TOKEN")
+  
```

Comment:
Contrary to other templates, the above templates doesn't include new-lines between the variables, which make it look messy and hard to read.

Reply:
done